### DISCUSSION TITLE
Questions about liabilities

### DISCUSSION THREAD

**Discussion by Bontah** [firefly-iii/firefly-iii#6347] [Firefly III] [answered]
Hello

i've read the documentation but still cannot understand the workflow for liabilities. other than that i'm really happy with firefly. incredible app!

1. Does it matter which "Type of liability" i pick? does the code work differently if i pick one or another type?
2. if i understand it correctly entries created under liabilities are basically just another form of accounts like assets, revenue and so on. is that correct?
3. assuming 2. is correct that means money exchange between friends, family and colleagues would mean i need create an account under the submenu liabilities to track everything? 
4. what if i lend a friend some money and later ask that same friend for some money too but that friend didnt repay me. do i create 2 accounts?
5. what if a friend asks me for money to buy some food, another time for books. i probably only add the amount to same account?
6. another thing i've been wondering: what about cash gifts i give or receice? how would i handle these?
7.  i created an account with a starting amount because i owe my friend some money but that is not on my any of my own account. does that mean i should instead create an account with 0 starting amount and just transfere money from that account to my own?
8. if i take a credit from my bank, lets say 5000€, my pay back rate is 250€/month but i need to pay 7k€ because of Interest. thats where the starting amount is used, right?

**Comment by JC5** [comment, marked as answer]
Heya,

1. No it doesn't really matter. It's for your administration only.
2. Yes, that's correct.
3. That would be easiest yes. Set it to "money owed to me".
4. You can just leave the liability open, so you can track it in the future.
5. Yes exactly, you can keep filling up that account. It's a little more complex though because you need two transactions. One to the liability itself, and then another one to the friend. See the explanation at the bottom.
6. If it's a gift then it's just a transaction, you don't really need to use liabilities. Just create a withdrawal or a deposit. The opposite account would not be a liability but a regular expense or revenue account.
7. Yes, if you didn't spend the money already. 
8. I would make the starting amount 5K. Every month, a new withdrawal to pay back 250 (from: your account, to: liability). Another transaction can be made for the interest (from: liability, to: bank (expense account)). 

For nr. 5. If you have a liability for what your friend is borrowing from you, and you wish to keep track of the money. Imagine that the liability is a bag of money your friend keeps taking money from. There needs to be money in the bag before he can take it.

1. Make a withdrawal from your bank account to the liability. This is money you are going to loan him. The liability will say after: balance 10, owed 0. There is 10 in the bag.
2. Send a second withdrawal from the liability to your friend. The liability will say: balance 0, owed 10. Your friend took money out of the bag.

If he pays you back, he puts 10 into the bag. If you want to use it for yourself, you must take it from the bag and put it into your own checking account.

**Reply by Bontah** [reply]
thanks that helped me alot!

**Reply by timaschew** [reply]
@JC5 
Is it really required to use two transactions over three accounts (your checking account, your liability account and your friends account)?

I don't see any value in it, why not just doing two transactions from your checking account to the liability account?

**Reply by JC5** [reply]
What's your question exactly?

**Reply by timaschew** [reply]
Maybe I miss something. Let me quote your sentence and make a question to it:

> Send a second withdrawal from the liability to your friend. The liability will say: balance 0, owed 10. Your friend took money out of the bag.

What does it really mean? 
What kind of account is it (the friends account), also a liability account?
Is this reflecting that the Bontah got money from his friend (back) or is the liability account just like a gate? But why do you need this?

**Reply by JC5** [reply]
You don't, technically speaking. I had a discussion about this but I can't remember the details.
